src/ucp/wireup/wireup.c

```diff
@@ -951,10 +951,12 @@ ucp_wireup_get_reachable_mds(ucp_worker_h worker,
 }
 
 ucs_status_t ucp_wireup_init_lanes(ucp_ep_h ep, unsigned ep_init_flags,
+                                   uint64_t local_tl_bitmap,
                                    const ucp_unpacked_address_t *remote_address,
                                    unsigned *addr_indices)
 {
     ucp_worker_h worker = ep->worker;
+    uint64_t tl_bitmap  = local_tl_bitmap & worker->context->tl_bitmap;
```

Comment:
I passed `UINT64_MAX` everywhere outside excepting CM cases like we usually do, then reduce bitmap here to reduce loop count in selection code.
As an alternative we can pass `worker->context->tl_bitmap` instead of `UINT64_MAX`

Reply:
ok let's keep with &